concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md

```diff
@@ -33,6 +33,72 @@ By default, the **ColorBox** does _not_ allow an end user to control the transpa
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            :value.sync="value"
```

Comment:
Let's call this variable 'color' to make it more meaningful. Apply here and below in Vue and React.

Reply:
fixed